wg-deployment/charter.md

```diff
@@ -0,0 +1,34 @@
+# WG Deployment Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+WG Deployment is responsible for tools to deploy apps from the catalog (e.g kfctl).
+
+### In scope
+
+#### Responsibilities
+
+- Provide tooling to deploy Kubeflow applications from the catalog.
+
+#### Code, Binaries and Services
+
+- kfctl (including operator)
```

Comment:
Is the operator part of `kfctl`?

Reply:
@jlewi, given that this is IBM/OpenShift specific, should it be part of upstream Kubeflow?